### ISSUE TITLE
0.23 docs build is broken

### ISSUE THREAD

**Issue by howardjohn** [rustls/rustls#1826]
Apologies if this was already reported somewhere. Seems the new build failed to build the docs: https://docs.rs/crate/rustls/0.23.0/builds/1143895

**Comment by cpu** [comment]
> Apologies if this was already reported somewhere.

It wasn't, so thank you for filing the issue! :-)

**Comment by ctz** [comment]
Yes, thank you for the report. Just released 0.23.1 that corrects this: https://docs.rs/crate/rustls/0.23.1
